test/parallel/test-process-title.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+const common = require('../common');
+const { spawnSync } = require('child_process');
+const { strictEqual } = require('assert');
+
+// FIXME add sunos support
+if (common.isSunOS)
+  common.skip(`Unsupported platform [${process.platform}]`);
+// FIXME add IBMi support
+if (common.isIBMi)
+  common.skip('Unsupported platform IBMi');
+
+const xs = 'x'.repeat(1024);
+const proc = spawnSync(process.execPath, ['-p', 'process.title', xs]);
+strictEqual(proc.stdout.toString().trim(), process.execPath);
```

Comment:
This gives an … odd … failure in Windows CI:

```
AssertionError [ERR_ASSERTION]: Expected values to be strictly equal:
+ actual - expected

+ 'npm'
- 'C:\\workspace\\node-test-binary-windows-js-suites\\node\\Release\\node.exe'
    at Object.<anonymous> (C:\workspace\node-test-binary-windows-js-suites\node\test\parallel\test-process-title.js:15:1)
```

(https://ci.nodejs.org/job/node-test-binary-windows-js-suites/1591/RUN_SUBSET=3,nodes=win2008r2-COMPILED_BY-vs2017/testReport/junit/(root)/test/parallel_test_process_title/)

Reply:
I believe I figured out what was causing that, see the fix-up commit.